numba_cuda/numba/cuda/codegen.py

```diff
@@ -26,10 +26,11 @@ def run_nvdisasm(cubin, flags):
     try:
         fd, fname = tempfile.mkstemp()
         with open(fname, "wb") as f:
-            if config.CUDA_USE_NVIDIA_BINDING:
+            # Support both ObjectCode (bindings/core) and raw bytes
+            if hasattr(cubin, "code"):
```

Comment:
This seems unnecessary. Why not just remove the branch instead of changing the code?